src/theme/components/Layout.tsx

```diff
@@ -47,6 +48,26 @@ const Layout = ({ children, title }: Props): JSX.Element => (
       <GlobalStyle />
       <Helmet title={title} />
       {children}
+      <CookieConsent
+        location="bottom"
+        cookieName={'ga-disable-G-D2D0FR22RN'}
+        enableDeclineButton={true}
+        flipButtons={true}
+        cookieValue={false}
+        declineCookieValue={true}
+        onDecline={() => {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```

Comment:
ℹ️ we could prevent to use this by adding a type definition on the global scope

Reply:
It's a good idea, but I don't know how to do that 😆

Reply:
done in 7c19ac4